bedrock/firefox/templates/firefox/features/adblocker.html

```diff
@@ -0,0 +1,216 @@
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="mzp-c-hero">
+    <div class="mzp-l-content l-content-narrow">
+      <div class="mzp-c-hero-body">
+        <h1 class="mzp-c-hero-title">{{ _('Why you might not use an ad blocker') }}</h1>
+        <div class="mzp-c-hero-desc">
+          <p>
+            {% trans %}
+            Many media and news organizations make most of their money from advertising. If you block ads on their sites, they won’t get paid. If enough people block ads, business could suffer, and so will the quality of reporting and writing. Luckily, most ad blockers let you make exceptions for your favorite sites.
+            {% endtrans %}
+          </p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="mzp-c-card-feature mzp-has-aspect-3-2 mzp-l-card-feature">
+    <div class="mzp-c-card-feature-media-wrapper">
+      {{ high_res_img('firefox/features/privacy-illustration2.png', {'alt': '', 'class': 'mzp-c-billboard-image'}) }}
+    </div>
+  </section>
+
+  <section class="mzp-c-hero">
+    <div class="mzp-l-content l-content-narrow">
+      <div class="mzp-c-hero-body">
+        <h1 class="mzp-c-hero-title">{{ _('How to set Tracking Protection on') }}</h1>
+        <p>{{ _('With Tracking Protection and an ad blocker installed, you’re getting a safe and speedy web experience. Here’s how to get started:') }}</p>
+        <ol class="mzp-u-list-styled">
+          <li>{{ _('Click the menu button and choose Options (or Preferences on a Mac)') }}.</li>
+          <li>{{ _('Click Privacy & Security on the left and go to the Tracking Protection section.') }}</li>
+          <li>{{_('Choose between the following options:') }}</li>
+          <ul class="mzp-u-list-styled">
+            <li>{{_('Always: Keep Tracking Protection on as you browse.') }}</li>
+            <li>{{_('Only in private windows: Turns Tracking Protection on only if you’re using Private Browsing.') }}</li>
+            <li>{{_('Never: Turns off Tracking Protection for all sites.') }}</li>
+          </ul>
+        </ol>
+        <p>
+          {% trans %}
+          Set Tracking Protection to always on and you won’t be reminded of that one time you looked at a trend pressure cooker. Pages will load faster, and you’ll have a better browsing experience.
+          {% endtrans %}
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <section class="mzp-l-content l-content-narrow">
+    <div class="mzp-c-hero-body">
+      <div class="mzp-c-hero-desc">
+        {{ high_res_img('firefox/features/laptop.png', {'alt': '', 'class': 'laptop'}) }}
+        <p>
+          {% trans %}
+          Set Tracking Protection to always on and you won’t be reminded of that one time you looked at a trendy pressure cooker. Pages will load faster, and you’ll have a better browsing experience.
+          {% endtrans %}
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <section class="mzp-c-call-out mzp-t-tracking">
```

Comment:
I don't think the theme class `mzp-t-tracking` exists in protocol, so I would probably drop the `mzp-` prefix as this is just bedrock specific styling. 

Unless this should be `mzp-t-firefox`?